### DISCUSSION TITLE
How to determine best dpi automatically when exporting pdf pages to jpg/png?

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#3861] [Ideas]
![image](https://github.com/user-attachments/assets/a2fb328f-2967-4bda-a03d-dc2c15bacfac)

**Comment by JorjMcKie** [comment]
No way to do that. Use 200 or 300 just to be sure.

**Reply by nissansz** [reply]
default value is 150?
I tried to ocr with exported image, sometimes, 300 dpi is better, sometimes 200 dpi is better, don't know how to determine which dpi to use
